### DISCUSSION TITLE
HOW.md - Low Level Implementation paragraph

### DISCUSSION THREAD

**Discussion by matteo-meluzzi** [HigherOrderCO/HVM1#112] [Q&A] [answered]
As mentioned in issue #74 it would be great if the Low-Level Implementation paragraph of the HOW.md https://github.com/Kindelia/HVM/blob/master/HOW.md#hvms-low-level-implementation would be completed.
Is anyone intentioned in doing it?

**Comment by steinerkelvin** [comment, marked as answer]
We are not doing that now, as we are busy with Kindelia/Kindelia (and Kindelia/Kind2).
